src/renderers/dom/fiber/__tests__/ReactDOMFiber-test.js

```diff
@@ -81,6 +81,21 @@ describe('ReactDOMFiber', () => {
   });
 
   if (ReactDOMFeatureFlags.useFiber) {
+
+    it('can import findDOMNode in Node environment.', () => {
+      const previousRAF = global.requestAnimationFrame;
+      const previousRIC = global.requestIdleCallback;
+      try {
+        global.requestAnimationFrame = undefined;
+        global.requestIdleCallback = undefined;
+        jest.resetModules();
+        require('ReactDOM');
```

Comment:
This is actually expected to throw since it determines you're in DOM mode. You need to keep this test (but change it to assert that it throws), and add another test that fools `canUseDOM` somehow to return `false` (look for existing tests that might do something similar—maybe it's enough to delete `window.document` or something like that).